sirepo/package_data/static/js/radia.js

```diff
@@ -1757,20 +1758,33 @@ SIREPO.app.directive('fieldDownload', function(appState, geometry, panelState, r
     };
 });
 
-SIREPO.app.directive('fieldLineoutReport', function(appState) {
+SIREPO.app.directive('fieldLineoutAnimation', function(appState, persistentSimulation, frameCache) {
     return {
         restrict: 'A',
         scope: {
-            modelName: '@'
+            modelName: '@',
         },
         template: `
             <div class="col-md-6">
-                <div data-ng-if="! dataCleared && hasPaths()" data-report-panel="parameter" data-request-priority="0" data-model-name="fieldLineoutReport"></div>
+              <div data-ng-if="! dataCleared && hasPaths()" data-report-panel="parameter" data-model-name="fieldLineoutAnimation">
+                <div data-ng-if="simState.isProcessing()" class="progress">
+                  <div class="progress-bar progress-bar-striped active" role="progressbar" aria-valuenow="{{ simState.getPercentComplete() }}" aria-valuemin="0" aria-valuemax="100" data-ng-attr-style="width: {{ simState.getPercentComplete() || 100 }}%"></div>
+                </div>
```

Comment:
I think you want a full simStatusPanel here, since we will want to run on nersc, select number of cores, etc.